src/js/pouchPersisted.js

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+/* global fluid, floe, PouchDB */
+
+(function ($, fluid) {
+
+    "use strict";
+
+    // Base grade for persistence of model components to Pouch
+    // Implementing grades should ensure that what's integral to
+    // reconstructing the particular instance is stored in the
+    // model
+    fluid.defaults("floe.dashboard.pouchPersisted", {
+        gradeNames: ["floe.dashboard.eventInTimeAware"],
```

Comment:
The core implementation file for this pattern is https://github.com/fluid-project/kettle/blob/master/lib/dataSource-core.js which is browser-ready as a result of @the-t-in-rtf 's suggestion and "just" needs to be hoisted up into core Infusion. It would be ok in the interim if you thought it was useful to start using this scaffold to fork this file into this project and cite https://issues.fluidproject.org/browse/FLUID-5900 for the rationalising work.